packages/route-viewer-overlay/src/index.tsx

```diff
@@ -127,9 +127,34 @@ const RouteViewerOverlay = (props: Props): JSX.Element => {
             reduceBounds,
             bounds || new LngLatBounds(coordsArray[0], coordsArray[0])
           );
+        } else if (geoJson?.type === "GeometryCollection") {
+          const { geometries } = geoJson;
```

Comment:
I don't think this is correct. Should it be `stop.geometries.geometries`? `geoJson` is coming from `stop.geometries` above

Reply:
Fixed it!

Reply:
I think we can do better than this! If this isn't always going to be a Polygon then why not update the type in the types package? This type is coming from the Stop type on line 502 of types.